src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
```

Comment:
it's better to use `span`
```suggestion
        <span> Posted by  </span>
```